auto-random.md

```diff
@@ -8,7 +8,7 @@ aliases: ['/docs/dev/auto-random/','/docs/dev/reference/sql/attributes/auto-rand
 
 > **Warning:**
 >
-> `AUTO_RANDOM` is still an experimental feature. It is **NOT** recommended that you use this attribute in the production environment. In later TiDB versions, the syntax or semantics of `AUTO_RANDOM` might change.
+> `AUTO_RANDOM` has been an official feature since v4.0.3. It is **recommended** that you use this attribute in the production environment. In later TiDB versions, the syntax or semantics of `AUTO_RANDOM` might change.
```

Comment:
It doesn't make sense that the syntax might change if it is now stable. May I suggest:
```suggestion
> `AUTO_RANDOM` was marked as stable in v4.0.3.
```